configure_machine/bootstrap_bevy_member_here.py

```diff
@@ -617,19 +650,23 @@ def get_salt_minion_id():
 
 def choose_master_address(host_name):
     default = host_name
-    if my_settings['master']:
+    if my_settings.get('master', ''):
         choices = get_ip_choices()
         print('This machine has the following IP addresses:')
         for ip in choices:
             if not ip['addr'].is_loopback and not ip['addr'].is_link_local:
                 print('{addr}/{prefix} - {name}'.format(**ip))
-    try:
-        # noinspection PyArgumentList
-        ip_ = socket.getaddrinfo(default, 4506, type=socket.SOCK_STREAM)
-        print('The name {} translates to {}'.format(host_name, ip_[0][4][0]))
-    except (socket.error, IndexError):
-        pass
-    while Ellipsis:  # repeat until user types a valid entry
+    if my_settings['master_host'] and default == "salt":
+        default = settings['vagrant_prefix'] + '.2.2'  # TODO: this will not work for IPv6

```

Comment:
Lack of IPv6 support makes Van Jacobson sad. But thank you for earmarking it for future improvement :-)